client/pages/checkout.js

```diff
@@ -225,6 +227,8 @@ export async function getServerSideProps(context) {
       },
     };
   }
+
+  return { props: {} };
```

Comment:
There is one error here because I forget to return empty props for the getServerSideProps function